test/include/sharg/tmp_filename.hpp

```diff
@@ -0,0 +1,151 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2021, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2021, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#pragma once
+
+/*
+ * Internal test infrastructure.
```

Comment:
I think this file should live in `test/` no? I think it also lives in the test include in seqan3.